vars/main.yml

```diff
@@ -12,6 +12,15 @@ jenkins_command: "/usr/bin/java {{ jenkins_java_args }} \
   --prefix={{ jenkins_prefix | quote }} \
   {{ jenkins_extra_args }}"
 
+# Delay (in seconds) to wait between retries when downloading Jenkins and related assets
+jenkins_download_delay: 3
```

Comment:
@nre-ableton In a677ebd65ed9c63d11609ff846aca68c0c8c9007, why was the delay changed from 5 to 3?

Reply:
Probably a copy/paste error. :sweat_smile: See https://github.com/Ableton/ansible-role-jenkins-jcasc/pull/252

Reply:
Thanks!